mockdata/api/accounts_mgmt/v1/subscriptions/3QChQVMVQx0CEoXuYZUMBQrQG8y.json

```diff
@@ -0,0 +1,156 @@
+{
+  "billing_expiration_date": "0001-01-01T00:00:00Z",
+  "capabilities": [
+    {
+      "inherited": false,
+      "name": "capability.cluster.manage_cluster_admin",
+      "value": "true"
+    }
+  ],
+  "cloud_account_id": "467305695994",
```

Comment:
refrain from using real account ids.  please replace it with a fake one.

Reply:
that was just an example.  please choose a different mock account id - `000000000005` already exists elsewhere

Reply:
Switched `000000000005` with `000000012345`
Hopefully that is ok